formfactory/templates/formfactory/form_detail.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <form class="Form" method="POST" action=".">
+    <form class="Form" method="POST" action="." enctype="multipart/form-data">
```

Comment:
Do a merge with develop please. Conditional csrf is in develop but I don't see it here yet.